js/main.js

```diff
@@ -24,4 +33,93 @@ function handleSubmitForm(event) {
   data.entries.unshift(formObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+
+  $entriesView.prepend(objectDOMTree(formObject));
+  $entryForm.classList.toggle('hidden');
+  $entriesView.classList.remove('hidden');
+  $newEntry.classList.toggle('hidden');
+  $entriesNewButton.classList.remove('hidden');
+  $entries.classList.toggle('hidden');
+  data.view = 'entries';
+}
+
+function objectDOMTree(formObject) {
+  var $ul = document.createElement('ul');
+  $ul.setAttribute('class', 'padding-left-0');
+
+  var $widthRow = document.createElement('div');
+  $widthRow.setAttribute('class', 'row width-row');
+
+  var $imageRow = document.createElement('div');
+  $imageRow.setAttribute('class', 'row image-row');
+
+  var $imgDOM = document.createElement('img');
+  $imgDOM.setAttribute('src', formObject.url);
+  $imgDOM.setAttribute('class', 'imageholder');
+  $imageRow.appendChild($imgDOM);
+  $widthRow.appendChild($imageRow);
+
+  var $columnHalf = document.createElement('div');
+  $columnHalf.setAttribute('class', 'column column-half');
+
+  var $h3 = document.createElement('h3');
+  $h3.setAttribute('class', 'label margin-left-20');
+  $h3.textContent = formObject.title;
+  $columnHalf.appendChild($h3);
+
+  var $notesP = document.createElement('p');
+  $notesP.setAttribute('class', 'info photo-url padding-left-20');
+  $notesP.textContent = formObject.notes;
+  $columnHalf.appendChild($notesP);
+
+  $widthRow.appendChild($columnHalf);
+  $ul.appendChild($widthRow);
+
+  return $ul;
+}
+
+function DOMObjectLoaded(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    $entriesView.appendChild(objectDOMTree(data.entries[i]));
+
+  }
+}
+
+var $newEntry = document.querySelector('.new-entry');
+var $noEntries = document.querySelector('.noEntries');
+
+function handleEntries(event) {
+  if (data.entries.length === 0) {
+    $noEntries.classList.remove('hidden');
+  }
+  $entryForm.classList.toggle('hidden');
+  $entriesView.classList.remove('hidden');
+  $newEntry.classList.toggle('hidden');
+  $entriesNewButton.classList.remove('hidden');
+  $entries.classList.toggle('hidden');
+  data.view = 'entries';
+}
+
+function handleNewEntry(event) {
+  if (data.entries.length === 0) {
+    $noEntries.classList.toggle('hidden');
+  }
+  $entriesNewButton.classList.toggle('hidden');
+  $entriesView.classList.toggle('hidden');
+  $entryForm.classList.remove('hidden');
+  $newEntry.classList.remove('hidden');
+  $entries.classList.remove('hidden');
+  data.view = 'entry-form';
+}
+
+window.addEventListener('DOMContentLoaded', sameOnLoad);
+
+function sameOnLoad(event) {
+  if (data.view === 'entries') {
+    $entryForm.classList.toggle('hidden');
+    $entriesView.classList.remove('hidden');
+    $newEntry.classList.toggle('hidden');
+    $entriesNewButton.classList.remove('hidden');
+    $entries.classList.toggle('hidden');
+  }
 }
```

Comment:
There's a lot of repetition here. Try to revise this to make a function for switching views that receives an argument of the name of the view to show, then shows that view and hides all other views. I recommend taking another look at `javascript-view-swapping`